content/docs/workflows/data-anonymization.md

```diff
@@ -1,182 +1,543 @@
+updatedOn: '2025-10-21T09:00:11.984Z'
 ---
 
-When working with production data, it's crucial to ensure that sensitive user information remains protected — especially in development or testing environments. With Neon, creating branches is fast, but how do you safely clone a production branch without exposing personal data?
+<FeatureBeta />
 
-The [PostgreSQL Anonymizer extension (`anon`)](/docs/extensions/postgresql-anonymizer) provides tools to mask, randomize, or obfuscate personal data, making it easy to create safe, anonymized branches for development and testing.
+Need to test against production data without exposing sensitive information? Anonymized branches let you create development copies with masked PII, emails, phone numbers, and other sensitive data.
 
-<Admonition type="important">
-Neon currently supports static masking with the `anon` extension, where anonymization rules are applied directly to the data in your branch. Note that the `anon` extension is experimental in Neon and requires explicit activation as shown below. Dynamic masking (on-the-fly anonymization during queries) is not yet available.
+Neon uses [PostgreSQL Anonymizer](https://postgresql-anonymizer.readthedocs.io/) for static data masking, and applies masking rules when you create or update the branch. This approach gives you realistic test data while protecting user privacy and supporting compliance requirements like GDPR.
+
+**Key characteristics:**
+
+- **Static masking**: Data is masked once during branch creation or when you rerun anonymization
+- **PostgreSQL Anonymizer integration**: Uses the [PostgreSQL Anonymizer extension's](/docs/extensions/postgresql-anonymizer) masking functions
+- **Branch-specific rules**: You can define different masking rules for each anonymized Neon branch
+
+<Admonition type="info" title="Static versus dynamic masking">
+This feature uses **static masking**, which permanently transforms data in the branch when anonymization runs. Unlike dynamic masking (which masks data during queries), static masking creates an actual masked copy of the data. To get fresh data from the parent, create a new anonymized branch.
 </Admonition>
 
-This guide demonstrates two approaches to anonymize data on a Neon branch:
+## Create a branch with anonymized data
 
-1. A manual procedure using SQL commands
-2. An automated process using GitHub Actions workflows
+<Tabs labels={["Console", "API"]}>
 
-## Anonymize branch data manually
+<TabItem>
 
-<Steps>
+Select **Anonymized data** as the data option when creating a new branch.
 
-## Prerequisites
+1. Navigate to your project in the Neon Console
+2. Select **Projects** -> **Branches** from the sidebar
+3. Click **New Branch**
+4. In the **Create new branch** dialog:
+   - Select your **Parent branch** (typically `production` or `main`)
+   - (Optional) Enter a **Branch name**
+   - (Optional) Set **Expire branch after** if you want to delete the branch automatically after a specified number of days
```

Comment:
We are going to update this text to "Automatically delete branch after" soon, so it'll be more clear what exactly happens to the branch.

Reply:
Understood, although that appears as part of a larger task (make all branches TTL 1 day) but once closer I'll update all instances in the docs.